cmake/Helpers.cmake

```diff
@@ -52,3 +52,22 @@ macro (install_symlink _filepath _sympath)
                     \"${_rel}\" \"\${_dst}\")
   ")
 endmacro (install_symlink)
+
+# For multi-config generators, the actual binary output path is not
+# `${EXECUTABLE_OUTPUT_PATH}`, but rather `${EXECUTABLE_OUTPUT_PATH}/$<CONFIG>`.
+# This macro creates a symlink to the actual target file in
+# `${EXECUTABLE_OUTPUT_PATH}`, such that the latest compiled build configuration
+# can be run using `./${EXECUTABLE_OUTPUT_PATH}/${_target}`.
+macro (multi_config_target_symlink _target)
+  get_property(_is_multi_config GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
+  if (_is_multi_config)
+    add_custom_command(
+      TARGET "${_target}"
+      POST_BUILD
+      COMMAND "${CMAKE_COMMAND}" -E create_symlink "$<TARGET_FILE:${_target}>"
+              "${EXECUTABLE_OUTPUT_PATH}/${_target}"
+      COMMENT
+        "Linking ${EXECUTABLE_OUTPUT_PATH}/${_target} -> $<TARGET_FILE:${_target}>"
```

Comment:
```suggestion
              "${EXECUTABLE_OUTPUT_PATH}/$<TARGET_FILE:${_target}>"
      COMMENT
        "Linking ${EXECUTABLE_OUTPUT_PATH}/$<TARGET_FILE:${_target}> -> $<TARGET_FILE:${_target}>"
```

Reply:
That doesn't make any sense, since the `TARGET_FILE` generator expression returns the absolute path to the target file.